src/basic/lines.js

```diff
@@ -0,0 +1,102 @@
+/*
+	 ___       ___  ________   _______   ________
+	|\  \     |\  \|\   ___  \|\  ___ \ |\   ____\
+	\ \  \    \ \  \ \  \\ \  \ \   __/|\ \  \___|_
+	 \ \  \    \ \  \ \  \\ \  \ \  \_|/_\ \_____  \
+	  \ \  \____\ \  \ \  \\ \  \ \  \_|\ \|____|\  \
+	   \ \_______\ \__\ \__\\ \__\ \_______\____\_\  \
+	    \|_______|\|__|\|__| \|__|\|_______|\_________\
+	                                       \|_________|
+
+ */
+
+/*
+	EXTERNAL DEPENDENCIES
+ */
+
+/*
+	INTERNAL DEPENDENCIES
+ */
+
+import BH from './helpers';
+import Utils from '../utils/utils';
+import TR from '../utils/reference';
+import Colors from '../style/colors';
+import Geom from '../utils/geom';
+
+const LR = TR.getLine(); // Line reference
+
+/*
+	INTERNAL LINE FUNCTIONS
+ */
+
+const getLineAlpha = function(c3ss) {
```

Comment:
compiled carto css => c3ss

It is the most repeated variable in the project, so it could be an important effort to write it as compiledCartoCSS 😓